message/association-setup-response.go

```diff
@@ -59,6 +59,8 @@ func NewAssociationSetupResponse(ies ...*ie.IE) *AssociationSetupResponse {
 			m.ClockDriftControlInformation = i
 		case ie.NFInstanceID:
 			m.UPFInstanceID = i
+		case ie.SequenceNumber:
```

Comment:
Please remove this.

Reply:
Will fix this and add fix for other node related messages as well.